src/validator-info.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+import * as BufferLayout from 'buffer-layout';
+import * as Layout from './layout';
+import {PublicKey} from './publickey';
+import {Account} from './account';
+import * as shortvec from './util/shortvec-encoding';
+import {struct} from 'superstruct';
+
+const VALIDATOR_INFO_KEY = new PublicKey('Va1idator1nfo111111111111111111111111111111');
+
+/**
+ * @private
+ */
+type ConfigKey = {|
+  publicKey: PublicKey,
+  isSigner: boolean,
+|};
+
+/**
+ * @private
```

Comment:
oh wait this isn't private.  Let's add esdoc comments so we get info in https://solana-labs.github.io/solana-web3.js/